packages/calcite-components/src/components/link/link.scss

```diff
@@ -56,13 +64,13 @@ calcite-icon {
 :host {
   span,
   a {
-    @apply text-color-link
-      transition-default
+    @apply transition-default
       relative
       inline
       border-none
       bg-transparent
       p-0;
+    color: var(--calcite-link-text-color, var(--calcite-color-text-link));
     line-height: inherit;
     white-space: initial;
     background-image: linear-gradient(currentColor, currentColor),
```

Comment:
For this underline, maybe we could pass it through with `--calcite-link-underline` ? Kind of a strange prop, I know design was maybe going to re-evaluate this, but for now we'd need to let users override it somehow. If they've overridden the link color they'll need to change this right?

Reply:
Lets just think about it. we can add it later.